randovania/games/blank/gui/preset_settings/__init__.py

```diff
@@ -16,4 +17,5 @@ def preset_tabs(editor: PresetEditor, window_manager: WindowManager):
         PresetItemPool,
         PresetBlankPatches,
         PresetDockRando,
+        PresetStartingArea
```

Comment:
I would not add this tab unconditionally until it's stable :)

Reply:
Is the blank game used for anything in stable?

Reply:
I'm blind. Thought I read Dread 🤣